lib/foreman_maintain/concerns/firewall/maintenance_mode.rb

```diff
@@ -0,0 +1,38 @@
+module ForemanMaintain
+  module Concerns
+    module Firewall
+      module MaintenanceMode
+        def notify_and_ask_to_install_firewall_utility
+          puts 'Unable to find nftables or iptables!'
+          question, pkg = question_and_pkg_name
+          answer = ask_decision(question, actions_msg: 'y(yes), q(quit)')
+          if answer == :yes
+            packages_action(:install, pkg)
+            feature(:instance).firewall.enable_maintenance_mode
+          end
+        end
+
+        def can_install_nft?
+          # The nftables is default from EL8 and Debian 10(Buster)
+          el_major_version >= 8 ||
+            deb_major_version >= 10 ||
+            ubuntu_major_version.to_i >= 22
```

Comment:
How does this work? On Debian `el_major_version` will return `nil` and that will fail:
```
[1] pry(main)> nil >= 8
NoMethodError: undefined method `>=' for nil:NilClass
from (pry):1:in `__pry__'
```